scripts/pi-hole/php/header.php

```diff
@@ -235,7 +235,7 @@ function pidofFTL()
             </a>
             <div class="navbar-custom-menu">
                 <ul class="nav navbar-nav">
-                    <!-- User Account: style can be found in dropdown.less -->
+                    <li class="dropdown"><a style="pointer-events:none;"><tt><?php echo gethostname(); ?></tt></a></li>
```

Comment:
[`<tt>` is deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt), use `<code>` instead.